main.rb

```diff
@@ -0,0 +1,67 @@
+require 'open3'
+
+def get_env_variable(key)
+  return (ENV[key] == nil || ENV[key] == "") ? nil : ENV[key]
+end
+ 
+def env_has_key(key)
+	value = get_env_variable(key)
+	return value unless value.nil? || value.empty?
+
+	abort("Input #{key} is missing.")
+end
+
+def run_command(command)
+	puts "@@[command] Executing: #{command}"
```

Comment:
Below is not a common usage as in other components.
<img width="243" alt="Screenshot 2024-11-12 at 16 10 34" src="https://github.com/user-attachments/assets/852ef487-cc06-4305-ba25-9396f29cd31e">

"Executing" should be removed, and the command log should have only the command itself.